doc/src/manual/noteworthy-differences.md

```diff
@@ -65,6 +65,8 @@ may trip up Julia users accustomed to MATLAB:
     similar to Python's bitwise operators (unlike C). They can operate on scalars or element-wise
     across arrays and can be used to combine logical arrays, but note the difference in order of operations:
     parentheses may be required (e.g., to select elements of `A` equal to 1 or 2 use `(A .== 1) .| (A .== 2)`).
+    Alternatively, the operators [`&&`](@ref) and [`||`](@ref) can be used to achieve control flow precedence
+    (similar to Matlab's `|` and `&`) and avoid the need for parentheses, as in `A .== 1 .|| A .== 2`.
```

Comment:
```suggestion
    similar to Python's bitwise operators (unlike C). For the logical boolean operators akin to Matlab's
    behavior, use Julia's `&&` and `||`, possibly in conjunction with broadcasting. For example, to test
    if some value `x` is less than `0` or greater than `1`, use `x < 0 || x > 1`, and to broadcast
    the equivalent test over an entire array `A`, use `A .< 0 .|| A .> 1`.
```

Reply:
How does this sound to you, @mestinso?

Reply:
Sounds good here, committed as suggested.